rocAL/rocAL/source/tf_meta_data_reader_detection.cpp

```diff
@@ -80,7 +80,6 @@ void TFMetaDataReaderDetection::lookup(const std::vector<std::string> &image_nam
             BoundingBoxCords bb_coords;
             BoundingBoxLabels bb_labels;
             BoundingBoxCord box;
-            ImgSizes img_sizes;
             ImgSize img_size;
 
```

Comment:
remove empty lines and cleanup the code to adjust spaces etc

Reply:
Done